rollup/internal/controller/sender/sender.go

```diff
@@ -618,13 +618,13 @@ func makeSidecar(blob *kzg4844.Blob) (*gethTypes.BlobTxSidecar, error) {
 	var commitments []kzg4844.Commitment
 	var proofs []kzg4844.Proof
 
-	for _, b := range blobs {
-		c, err := kzg4844.BlobToCommitment(b)
+	for i := range blobs {
+		c, err := kzg4844.BlobToCommitment(&blobs[i])
```

Comment:
isn't it equivalent to the previous?

Reply:
the previous implementation would get golint error as follows:
```
internal/controller/sender/sender.go:622:38: G601: Implicit memory aliasing in for loop. (gosec)
                c, err := kzg4844.BlobToCommitment(&b)
                                                   ^
internal/controller/sender/sender.go:627:38: G601: Implicit memory aliasing in for loop. (gosec)
                p, err := kzg4844.ComputeBlobProof(&b, c)
                                                   ^
```